config/environments/production.rb

```diff
@@ -118,4 +118,15 @@
     'X-Content-Type-Options' => 'nosniff',
     'X-XSS-Protection'       => '1; mode=block',
   }
+
+  if ENV['SLACK_NOTITIFER_URL'] && ENV['SLACK_NOTIFIER_CHANNEL']
```

Comment:
How about to add condition for `ENV['SLACK_NOTIFIER_USER']`? I think `ENV['SLACK_NOTIFIER_USER']` is needed for condition(if). because middleware's configuratino is using `ENV['SLACK_NOTIFIER_USER']`.

e.g.
```
if ENV['SLACK_NOTITIFER_URL'] && ENV['SLACK_NOTIFIER_CHANNEL'] && ENV['SLACK_NOTIFIER_USER']
```